bots/image-refresh

```diff
@@ -86,15 +86,20 @@ TRIGGERS = {
         "fedora-30/selenium-chrome",
         "fedora-30/selenium-firefox",
     ],
+    "rhel-7-6": [
+        "rhel-7-6@cockpit-project/cockpit/rhel-7.6",
+    ],
     "rhel-7-7": [
         "rhel-7-7/firefox@weldr/cockpit-composer",
+        "rhel-7-7@cockpit-project/cockpit/rhel-7.7",
     ],
     "rhel-8-0": [
-        "rhel-8-0",
```

Comment:
Technically we also still support master on rhel-8-0. Practically that's not an interesting target any more, but we should make such policy decisions/changes outside of infrastructure PRs like this. So let's please put back the original rhel-8-0 context.